applications/Chat/evaluate/requirements.txt

```diff
@@ -0,0 +1,10 @@
+jieba
+bert-score
+rouge_chinese
+scikit-metrics
+nltk
+openai
+seaborn
+pandas
+matplotlib
+numpy
```

Comment:
We do not have additional `setup.py` for evaluation. So adding `requirements.txt` here can not install all these packages.

All these should be moved back to `applications/Chat/requirements.txt` as the `setup.py` will fetch this file there.

Reply:
I assume what we have done before was wrong? If you check the `setup.py` file it only fetches the `requirements.txt` from the same directory.

Any comments or my understanding is wrong? @ver217

Reply:
`requirements.txt` is essential to use `coati`. `examples/requirements.txt` is like extra requirements to run examples. I think it's OK. As for inference, the installation process of GPTQ is very difficult and users must follow its read me then install it manually.

As for evaluation, do you think it's a part of coati? If so, move the evaluation directory to `coati/` and add requirements to `requirements.txt`. Otherwise, keep them apart.

Reply:
`coati` is regarded as a library. So if evaluation is a part of `coati`, you'd better provide CLI.

Reply:
Ok, sure.

We can keep it separate, but make sure in the readme file, you will need to add a setup section for `pip install -r requirements.txt` 

@chengeharrison @Camille7777

Reply:
Ok.

Reply:
noted